docs/standard/choosing-core-framework-server.md

```diff
@@ -0,0 +1,106 @@
+
+## When to choose .NET Framework
+
+While .NET Core offers significant benefits for new application patterns, the .NET Framework will continue to be the natural choice for many existing scenarios and as such, it won’t be replaced by .NET Core for all server applications. In general, you should consider using .NET Core because of the benefits provided by it, except in the following cases where .NET Framework is a better match:
+
+### Current .NET Framework applications
+
+In most cases, you won’t need to migrate your existing applications to .NET Core. Instead of that, the natural approach is to embrace .NET Core as you extend your existing applications, such as adopting microservices for new additions to your system. The .NET Framework will continue to be shipped at its current pace, sharing the innovation in areas like the .NET Standard Library, languages, and core components. .NET Framework constraints solved by .NET Core such as cross-platform, modularity, and performance don’t affect existing applications in traditional workloads. You should evaluate .NET Core only if these benefits outperform the cost of a migration.
+
+### A need to use third party .NET libraries or NuGet packages not available for .NET Core
+
+Libraries are quickly embracing .NET Standard, which enables sharing code across all .NET flavors including .NET Core. With .NET Standard 2.0 this will be even easier, as the .NET Core API surface will become significantly bigger and .NET Core applications can directly use existing .NET Framework libraries. This transition won’t be immediate, though, so we recommend checking the specific libraries required by your application before making a decision one way or another.
+
+### A need to use .NET technologies not available for .NET Core
+
+Some .NET Framework technologies are not available in .NET Core. Some of them will be available in later .NET Core releases, but others don’t apply to the new application patterns targeted by .NET Core and may never be available. The following list shows the most common technologies not found in .NET Core 1.0:
+
+* ASP.NET Web Forms applications: ASP.NET Web Forms is only provided by the .NET Framework, so you cannot use ASP.NET Core / .NET Core for this scenario. Currently there are no plans to bring ASP.NET Web Forms to .NET Core.
+
+* ASP.NET Web Pages applications: At .NET Core 1.0 release timeframe (June 2016), ASP.NET Web Pages are not included in ASP.NET Core, although it is planned to be included in a future release as explained in the [.NET Core roadmap](https://github.com/aspnet/Home/wiki/Roadmap).
```

Comment:
Feels weird to mention these dates and releases like this "At .NET Core 1.0 release timeframe (June 2016)" (nultiple instances)

Suggested rewriting:
ASP.NET Web Pages are not included in ASP.NET Core 1.0 , although we're planning to include them...